packages/react-front-kit-shared/src/helpers/typeGuard.ts

```diff
@@ -26,3 +27,18 @@ export function typeGuard<T extends IPrimitiveOrConstructor>(
   }
   return o instanceof localPrimitiveOrConstructor;
 }
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ITypeGenericInterface = Record<string, any>;
```

Comment:
Pas possible d'utiliser `unknown` plutôt que `any` pour éviter le commentaire ?

Reply:
sans le any, donner un type à la la fonction cause une erreur (par exemple dans le fichier de test) : 
`TS2344: Type  ITest  does not satisfy the constraint  ITypeGenericInterface 
Index signature for type  string  is missing in type  ITest`

Reply:
ok